packages/react-native-renderer/src/ReactNativeFrameScheduling.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
We should probably call this something other than "FrameScheduling" since the plan is no longer to attempt to line up with Frames per se.

Reply:
I didn't think too hard about the name since we plan to change it soon anyway. I'll rename it when we do the real implementation.